### ISSUE TITLE
--ifempty not working

### ISSUE THREAD

**Issue by elint** [FezVrasta/ark-server-tools#482]
should be "if (( numplayers != 0 ));" in the below code:

```
      elif [ "$updatetype" == "ifempty" ]; then
        numplayers=$(( $(numPlayersConnected) + 0 ))
        if (( numplayers == 0 )); then
          echo "${numplayers} players are still connected"
          return 1
        fi
```

Tried setting a cron job using ifempty, and it updated and booted a player out while he was on the server.  I'm assuming the above code is incorrectly saying to only update WHEN a user is connected.

cheers.

**Comment by klightspeed** [comment]
Please try the `1.6-dev.unstable` branch.

This issue was fixed in 63ded49 (in #472)

**Comment by elint** [comment]
Aah.  I was actually submitting this to the original repository that FezVrasta runs for arkmanager.  I would think this should be patched in the main branch of that software, too.  As for your unstable branch, it certainly looks promising.  I'm not used to seeing a random mishmash of bugfixes all bundled up in the same pull request, so I'll have to review the whole thing before deciding if it's worth applying all those other changes to fix this one small bug.  In the meantime, hopefully this can be patched in the main branch, too :)

cheers.

**Comment by klightspeed** [comment]
This should be fixed in the hotfix to v1.5
